### DISCUSSION TITLE
ToDo 진행 상황

### DISCUSSION THREAD

**Discussion by InChil2** [InChil2/SlotTagging#46] [코드 Q&A]
1107 ToDO []

**Comment by hharimjung** [comment]
1109 ToDO

**Reply by hharimjung** [reply]
* 시나리오 if 구문 작성
* 피그마 css, html 코드 수정

**Reply by hharimjung** [reply]
- [x] (코드) size 슬롯에 grande 추가하기

- [x] (코드) beverage 슬롯에  ‘아이스’ 텍스트 삭제(온도로 별도 구별했음)

- [x] (PPT) FIGMA, CSS, HTML 기술 정의 정리하기

- [x] (코드) 챗봇 시나리오 출력문구 수정하기

- [x] (웹구현/시각화) 시각화 아이콘 문제 해결
